tests/PactNet.Native.Tests/Verifier/NativePactVerifierPairTests.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using Moq;
+using PactNet.Native.Verifier;
+using PactNet.Verifier;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PactNet.Native.Tests.Verifier
+{
+    public class NativePactVerifierPairTests
+    {
+        private readonly NativePactVerifierPair verifier;
+
+        private readonly Mock<IVerifierProvider> mockProvider;
+
+        private readonly IDictionary<string, string> verifierArgs;
+
+        public NativePactVerifierPairTests(ITestOutputHelper output)
+        {
+            this.mockProvider = new Mock<IVerifierProvider>();
+
+            this.verifierArgs = new Dictionary<string, string>();
+
+            this.verifier = new NativePactVerifierPair(this.verifierArgs,
+                                                       this.mockProvider.Object,
+                                                       new PactVerifierConfig
```

Comment:
To fix the formatting, maybe add a variable to contain the config or like in your NativePactVerifierProviderTests test passing a newline.